cluster-stamp.bicep

```
@@ -1558,7 +1546,7 @@ resource kvPodMiIngressControllerSecretsUserRole_roleAssignment 'Microsoft.Autho
   }
 }
 
-// Grant the AKS cluster ingress controller pod managed identity with key vault reader role permissions; this allows our ingress controller to pull certificates
+// Grant the AKS cluster ingress controller managed workload identity with key vault reader role permissions; this allows our ingress controller to pull certificates
```

For consistency with above:
```suggestion
// Grant the AKS cluster ingress controller's managed workload identity with key vault reader role permissions; this allows our ingress controller to pull certificates.
```

---

Good catch

---

Resolved in: https://github.com/mspnp/aks-baseline/pull/326/commits/3f3058684316dffe84b7b6180ccbf5a212c27a94